backend/pkg/config/kafka_sasl_oauth.go

```diff
@@ -10,25 +10,95 @@
 package config
 
 import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
 	"flag"
 	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
 )
 
 // KafkaSASLOAuthBearer is the config struct for the SASL OAuthBearer mechanism
 type KafkaSASLOAuthBearer struct {
-	Token string `yaml:"token"`
+	Token         string `yaml:"token"`
+	ClientID      string `yaml:"clientid"`
+	ClientSecret  string `yaml:"clientsecret"`
```

Comment:
```suggestion
	ClientID      string `yaml:"clientId"`
	ClientSecret  string `yaml:"clientSecret"`
```

For consistency, lower camel case